phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
All right, sounds like it's empirical. If you can prove to me that getting
a channel's LFP is nearly as fast from the raw data as from being read from
a dedicated file, then fine :)

On Tue, Jun 30, 2015 at 7:10 PM, Cyrille Rossant <notifications@github.com>
wrote:

> In phy/cluster/algorithms.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33604503>:
>
> > @@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
> >      def _create_filter(self):
> >          rate = self._kwargs['sample_rate']
> >          low = self._kwargs['filter_low']
>
>  High-pass filtered data aren't saved, they're simply processed and
> features extracted as needed. Waveforms are extracted, filtered and aligned
> in realtime every time you load a cluster.
>
> Actually, these *are* all saved in the temporary store, notably for
> performance reasons. All extracted waveforms are saved on disk during spike
> detection, and, currently, they stay here until you delete them. In the
> GUI, the subset of waveforms of every cluster are also saved when you first
> load your dataset in phy.
>
> Regarding LFP, I'd rather save them *during* spike detection actually,
> because we can benefit from the fact that the raw data is already loaded in
> RAM. This will also be nicely parallelized once SD is parallelized.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33604503>.
>